packages/backend/src/App.ts

```diff
@@ -279,9 +282,24 @@ export default class App {
             );
         }
 
-        await this.serviceRepository
-            .getOrganizationService()
-            .initializeInstance();
+        try {
+            const instanceConfigurationService =
+                this.serviceRepository.getInstanceConfigurationService<InstanceConfigurationService>();
+            await instanceConfigurationService.initializeInstance();
+        } catch (e) {
+            if (e instanceof MissingConfigError) {
+                console.debug(
+                    `No instance configuration service found: ${getErrorMessage(
+                        e,
+                    )}`,
+                );
+            } else {
+                Logger.error(
+                    `Unable to initialize instance ${getErrorMessage(e)}`,
+                );
```

Comment:
if an error happens during initialization, we want to stop the app, something was wrong and it needs to be fixed

Reply:
Tiny thing, but do we need to log and throw the error? Would just throwing an error suffice?